### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by Kristin-ZLCHEN** [yulequan/HeartSeg#10]
Hello,
I am fused in why you choose softmaxWithLoss as Loss Function? It is related to the training label?
